torch/_higher_order_ops/triton_kernel_wrap.py

```diff
@@ -239,25 +248,49 @@ def convert(token):
     # In alternative representation, function names are quoted.
     # It should be possible to move this into the grammar alltogether.
     def convert_name(token):
+        if token is None:
+            return None
```

Comment:
When can `token` be `None`? Maybe add a comment?

Reply:
When a token is optional, it can be null. I can add something.